Проверяем рабочий процесс

vshirmanov opened the pull request:
---
:mortar_board: [Проверяем рабочий процесс](https://up.htmlacademy.ru/htmlcss/33/user/1711721/tasks/2)

:boom: https://htmlacademy-htmlcss.github.io/1711721-sedona-33/1/

vshirmanov committed:
Добавляет имя наставника

LeMarck commented on Readme.md:
```diff
@@ -1,7 +1,7 @@
 
 * Студент: [Василий Ширманов](https://up.htmlacademy.ru/htmlcss/33/user/1711721).
-* Наставник: `Неизвестно`.
+* Наставник: `Евгений Петров`.
```
Давай сделаем по аналогии с полем "Студент"

```md
[Евгений Петров](https://htmlacademy.ru/profile/id617077)
```

vshirmanov replied:
Готово!

vshirmanov committed:
Update Readme.md

Оборачивает имя в квадратные скобки и добавляет ссылку на профиль наставника на htmlacademy.

vshirmanov reviewed (commented):
Готово!
